### PR TITLE
Add Pipfile and instructions

### PR DESCRIPTION
**Pull request by txtsd**
Adds a `Pipfile` so we can use `pipenv` to setup a vritual environment instead of `pip install --user <package>`
Also adds a `requirements.txt` in case someone wants to use another virtualenv tool or just plain `pip`.

Additionally, fixes a typo and a list's formatting.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+requests = "*"
```
Comment (bugwelle) [inline Pipfile:7]: I'm not familiar with Pipfile. Why is `requests = *` here but in `requirements.txt` something different? :-)

**Review by bugwelle — COMMENTED** [review]
Cool! Thank you for contributing!

File: Pipfile (hunk 1 above)
Comment (txtsd) [reply Pipfile:7]: `Pipfile` is what `pipenv` uses.
`requirements.txt` doesn't take a wildcard so it needs to be a version number. I've only included it as a fallback.
Would you like `Pipfile` to have the same version constraint as `requirements.txt`?

Comment (bugwelle) [reply Pipfile:7]: Thanks! No, that's not necessary. This PR looks good to me. 👍

**Review by bugwelle — APPROVED** [review]

**Pull request merged**